### PR TITLE
Module 10

### PR DESCRIPTION
**Pull request by juasmartinezbelS4N**

### CODE REVIEW

**Comment by MinMiguelM** [comment]
Please let me know when it is completed, thanks!

File: src/module_10/database/mongoDB.ts [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
 import * as ProductEntities from "./schemas/product.entity"
-import User from "./documents/user.document"
 import Product from "./documents/product.document"
-import 'dotenv/config'
-
+import { logger } from "../logger"
+import "dotenv/config"
 
 const startMongo = () => {
   const uri: string = `mongodb://localhost:${process.env.DB_PORT}`
```
Comment (MinMiguelM) [inline src/module_10/database/mongoDB.ts]: Host should be in a env variable

**Review by MinMiguelM — CHANGES_REQUESTED** [review]
One small change, almost done!

File: src/module_10/database/mongoDB.ts (hunk 1 above)
Comment (juasmartinezbelS4N) [reply src/module_10/database/mongoDB.ts]: I DON'T KNOW HOW I MISSED THIS, THANK YOU!

**Review by MinMiguelM — APPROVED** [review]
Well done

**Pull request merged**